### PR TITLE
Go/Svelte automation panel

### PR DESCRIPTION
**Pull request by hellt**

### CODE REVIEW

**Comment by hellt** [comment]
Hi @tiago-amado @mairp @aaakpinar 

please try this branch, it has an improved automation panel that is based on a modern framework that we can reuse for other labs easily now.

Let me know if that works on your end and then I will merge it

File: automation_panel/backend/gnmi.go [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+type updateResultMsg struct {
+	Operation string `json:"operation,omitempty"`
+	Path      string `json:"path,omitempty"`
+	Target    string `json:"target,omitempty"`
+}
+
+func getValue(updValue *gnmi.TypedValue) (interface{}, error) {
+	if updValue == nil {
+		return nil, nil
+	}
```
Comment (hellt) [inline automation_panel/backend/gnmi.go:59]: @siva19susi I believe none of this should be part of the backend
gnmic exposes the API that hides all this https://gnmic.openconfig.net/user_guide/golang_package/examples/get/

Comment (siva19susi) [reply automation_panel/backend/gnmi.go:59]: @hellt the get and set is using the same code mentioned in that example link you shared but the way to format the response is not defined (rather just prototext). How do I consume the below json.go file as a package and its functions ? 

https://github.com/openconfig/gnmic/blob/30c1330e5d37da7f05b419627cddf4479f412068/pkg/formatters/json.go#L223

Comment (hellt) [reply automation_panel/backend/gnmi.go:59]: that's fine. Leave this with me, I will work on some polishing over this PR, but this one I think we are good to merge

**Pull request merged**